packages/web/src/components/edit-collection/desktop/EditCollectionForm.tsx

```diff
@@ -21,14 +17,13 @@ import {
 } from 'components/form-fields'
 import { Tile } from 'components/tile'
 
-import { AnchoredSubmitRow } from '../components/AnchoredSubmitRow'
-import { AccessAndSaleField } from '../fields/AccessAndSaleField'
-import { CollectionTrackFieldArray } from '../fields/CollectionTrackFieldArray'
-import { ReleaseDateFieldLegacy } from '../fields/ReleaseDateFieldLegacy'
-import { SelectGenreField } from '../fields/SelectGenreField'
-import { SelectMoodField } from '../fields/SelectMoodField'
-import { StemsAndDownloadsCollectionField } from '../fields/StemsAndDownloadsCollectionsField'
-import { CollectionFormState } from '../types'
+import { AnchoredSubmitRow } from '../../../pages/upload-page/components/AnchoredSubmitRow'
```

Comment:
i think this can actually be `import { AccessAndSaleField } from 'components/edit/fields/AccessAndSaleField'`?